pynhm/base/control.py

```diff
@@ -28,6 +30,7 @@ def __init__(
         time_step: np.timedelta64,
         init_time: np.datetime64 = None,
         config: dict = None,
+        params: PrmsParameters = None,
```

Comment:
potential controversy
It kinda makes sense to me to have all the runtime inputs/options available in one place, in the controller.